### ISSUE TITLE
PROPOSAL: Move to git-commit-id organization

### ISSUE THREAD

**Issue by ktoso** [git-commit-id/maven-git-commit-id-plugin#384]
Hey @TheSnoozer,
I was thinking about this recently. Since I've not really maintained this plugin and you are doing a great job here, and perhaps others may want to join in as well.

I was thinking of moving the plugin to https://github.com/git-commit-id as I guess https://github.com/git-commit-id/maven-plugin perhaps, and ofc invite you as admin as well.

It would make the project feel more "free" as in freedom rather than remaining under my account while I don't really actively maintain it since quite a while.

Do you think this sounds good?

**Comment by TheSnoozer** [comment]
Mhhh....for me personally I don't see how anything is "not free" while it is under your github account. You invented the project at some point and thus keeping it attached to your name refers (more) to you as the original author/inventor. If you feel you want to "free" yourself from this somewhat "only maintained" project, sure let's do that. 

I personally don't see any huge benefit by moving it to a different github user account, since unfortunately there is not a lot (other) user contribution and I feel having it under a different github user won't change that drastically.

So in short, for me personally I stopped using maven ages ago and I'm just here to keep this project in a use-able state. If you think "freeing" would help in any way, sure let's do it. If so I'd propose to use https://github.com/git-commit-id/git-commit-id and keep any name reference to maven (for the repository name) out of the game since at some point there might actually be a plugin for gradle...

**Comment by ktoso** [comment]
> So in short, for me personally I stopped using maven ages ago

Yeah same for me lol, so I figured maybe it's healthier to signal this via the repo move. Even if we'd continue our best effort as we do nowadays...

I like your naming proposal for the repos.

I think I'll ponder this a bit more and ping here once more. I think I would be fine with moving it; perhaps it will signal that it's looking for more maintainers who actively use it and maven as well then perhaps.

Thanks for being open and so nice about giving me credits :) I'll ponder some more but am thinking it could have a positive impact on anyone who's really using it... I'll keep in touch!

**Comment by TheSnoozer** [comment]
> Thanks for being open and so nice about giving me credits :) 
Honestly, you deserve just that. It was a really great plugin for the time I used it and I think it is still usable for those who still use maven. However I was truly annoyed by being forced to think in the maven way. If you want to do anything that is not considered ``the maven way`` you end up writing 300+ lines of xml hackish code that becomes unmaintainable because it's not officially supported. Just try to add the git abbrev to the artifact you want to publish.....in gradle this is about 5 lines of code including try catches for wierdo errors.

Regardless....I also would throw in another thought that somewhat popped up in my mind and make stuff more aligned with the move.
As of now the plugin is known as
```
<groupId>pl.project13.maven</groupId>
<artifactId>git-commit-id-plugin</artifactId>
```
Shall we change that too?
```
<groupId>git-commit-id-plugin</groupId>
<artifactId>git-commit-id-plugin</artifactId>
```
?

AFAIK and unfortunately that might mean that users never will know that the plugin moved to a different repo.

**Comment by ktoso** [comment]
yeah group name changes are more painful... to that extent that we never changed them at typesafe=>lightbend even. So while code moved from github.com/typesafehub/config => github.com/lightbend/config and even Akka itself still releases under `com.typesafe` (!) years after the company did the rename...

It is mostly for the discoverability issue but also a runtime dependency hell thing (matters more for libraries, not really for us in the plugin here...). But at least for now I think we'd not change the group ID hm... We maybe could do so once we do the Gradle version, and then mage it a "major version bump™" changing group ids along the way hm

**Comment by TheSnoozer** [comment]
Let's agree to change the version "later™", which in all fairness should do ;-)

**Comment by TheSnoozer** [comment]
Alternatively name it 
https://github.com/git-commit-id-plugin/git-commit-id-plugin

in the end it will always be a plugin.
I wouldn't mind and doesn't make a huge difference...

**Comment by TheSnoozer** [comment]
@ktoso you started this yet (https://github.com/git-commit-id) ?

otherwise I'll just get this rolling ;)

**Comment by ktoso** [comment]
Hey yeah, I think we should make this happen as I'm unavailable for participation here for a while...
Can you remind me your email address ping me at `ktoso @ project13 pl`

**Comment by TheSnoozer** [comment]
Thanks @ktoso for taking care of that :-)

The old repo
https://github.com/ktoso/maven-git-commit-id-plugin
also automatically redirects to
https://github.com/git-commit-id/maven-git-commit-id-plugin

closing as solved :-)

**Comment by ktoso** [comment]
Thanks a lot for all your help @TheSnoozer, you also have admin powers there :)

Note: purely administrative task here, not a contribution :)
